server/controllers/report.js

```diff
@@ -158,10 +166,74 @@ function deleteArchived(req, res, next) {
     .done();
 }
 
+// TODO(@jniles) - translate these emails into multiple languages
+const REPORT_EMAIL =
+`Hello!
+
+Please find the attached report "%filename%" produced by %user% on %date%.
+
+This email was requested by %requestor%.
+
+Thanks,
+bhi.ma
+`;
+
+// this is a really quick and lazy templating scheme
+const template = (str, values) => {
+  return Object.keys(values).reduce((formatted, key) =>
+    formatted.replace(`%${key}%`, values[key]), str);
+};
+
+
+/**
+ * @function emailArchived
+ *
+ * @description
+ * Emails an archived report to an email address provided in the "to" field.
+ */
+function emailArchived(req, res, next) {
+  const { uuid } = req.params;
+  const { address } = req.body;
+
+  debug(`#emailArchived(): Received email request for ${address}.`);
+
+  lookupArchivedReport(uuid)
+    .then(report => {
+      debug(`#emailArchived(): sending ${report.label} to ${address}.`);
+
+      const date = moment(report.timestamp).format('YYYY-MM-DD');
+      const filename = `${report.label}.pdf`;
+
+      const attachments = [
+        { filename, path : report.link },
+      ];
+
+      // template parameters for the email
+      const parameters = {
+        filename,
+        date,
+        user : report.display_name,
```

Comment:
What is the difference between the user and the requester?

Reply:
The user created the report originally and archived it.  The requester is the person asking for the email to be sent.   The AG might create the report, but the MD might request it to be sent somewhere.